doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -38,7 +38,7 @@ IDE and tool support
 Build and configuration system
 ==============================
 
-|no_changes_yet_note|
+* ``SB_CONFIG_MCUBOOT_USE_ALL_AVAILABLE_RAM`` Kconfig option added to system to allow using all available RAM when using TF-M on nRF5340. Note that this has security implications and may allow secrets to be leaked to the non-secure application in RAM.
```

Comment:
```suggestion
* Added the ``SB_CONFIG_MCUBOOT_USE_ALL_AVAILABLE_RAM`` Kconfig option to system to allow using all available RAM when using TF-M on an nRF5340 device.

  .. note:: 
     This has security implications and may allow secrets to be leaked to the non-secure application in RAM.
```

Reply:
Fixed

Reply:
Oh, noted some redundancy as well, using ... when using. Maybe something like "to allow using all available RAM with TF-M on..."

Reply:
Have updated the word